README.md

```diff
@@ -105,7 +105,10 @@ sampler.pause()
 sampler.resume()
 
 # Wait for the sampler to finish (up to timeout seconds)
-# sampler.wait(timeout=0.1)
+sampler.wait(timeout=0.1)
+# Note that not passing any timeout to `wait` will
+# directly save and return the current results once the sampler finishes:
```

Comment:
Apologies, that was actually a legit question, I think I was confused by "directly".
```suggestion
# block until the sampler finishes and return the InferenceData object:
```